lib/nunes/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Nunes
-  VERSION = '0.6.0'
+  VERSION = '0.6.1'
```

Comment:
Curious.. why did the version get bumped again even though it hasn't been released?

Reply:
It was easier to get Rails to pull latest on my machine this way. Other than that, no reason.